server/views/pages/bookVisit/checkVisitDetails.njk

```diff
@@ -1,22 +1,153 @@
+{% extends "../../partials/layout.njk" %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
-<pre>
-  <h1>Check your booking</h1>
-
-  BOOKER
-  {{ booker | dump(2) }}
-
-  BOOKING JOURNEY
-  {{ bookingJourney | dump(2) }}
-  
-</pre>
-
-<form action="/book-visit/check-visit-details" method="POST" novalidate>
-  <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-
-  {{ govukButton({
-    text: "Submit booking",
-    classes: "govuk-!-margin-top-3",
-    preventDoubleClick: true,
-    attributes: { "data-test": "submit-booking" }
-  }) }}
-</form>
\ No newline at end of file
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% set pageTitle = "Check the visit details before booking" %}
+
+{% set backLinkHref = "/book-visit/main-contact" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">Check the visit details before booking</h1>
+
+      {% include "partials/errorSummary.njk" %}
+
+      {% set additionalSupportRows %}
+      {% if additionalSupport | length %}{{ additionalSupport }}{% else %}None{% endif %}
+      {% endset %}
```

Comment:
This looks like a hangover from when additional support was more than just a text field. Suggest removing this and changing line 97 to:
```
html: '<p class="test-additional-support">' + (additionalSupport if additionalSupport | length else "None") + '</p>'
```